tests/kafkatest/services/performance/templates/tools_log4j.properties

```diff
@@ -19,7 +19,7 @@ log4j.rootLogger = {{ log_level|default("INFO") }}, FILE
 log4j.appender.FILE=org.apache.log4j.FileAppender
 log4j.appender.FILE.File={{ log_file }}
 log4j.appender.FILE.ImmediateFlush=true
-# Set the append to false, overwrite
-log4j.appender.FILE.Append=false
+# Set the append to true, append
+log4j.appender.FILE.Append=true
```

Comment:
Why was this needed? Append is normal for long running services that we need to bounce since we want to capture all the output, but this seems unusual for tools. They are usually one-off. I don't think there's necessarily anything wrong with this, but I'd like to understand the reasoning since this template is used across 5 or 6 separate tools.